### ISSUE TITLE
Store history in browser storage

### ISSUE THREAD

**Issue by rouge8** [sosedoff/pgweb#706]
I would like to be able to store the query history in the History tab in browser storage instead of in the server memory. This would make it per-user and persist after restarting pgweb.

**Comment by sosedoff** [comment]
If you're concerned about history leaking across users, you can run pgweb in multi-session mode with `--sessions` flag.

**Comment by rouge8** [comment]
Oh nice, I'll do that. It'd still be nice to save in browser history so it persists across restarts though.

**Comment by sosedoff** [comment]
Yea this kind of request has been on the radar for a while, but i dont have any ETA for when its going to be available.

**Comment by rouge8** [comment]
Yeah, I understand. Also thanks for pgweb! It's exactly what I was looking for 😀

**Comment by rouge8** [comment]
Oh, it looks like setting `PGWEB_SESSIONS=1` is for multiple database connections, so it ignores the `--url` option...
